File: 프로그래머스/0/120903. 배열의 유사도/배열의 유사도.js
function solution(s1, s2) {
    var answer = 0;
    let repo = new Set();
    for (let s of s1) {
        repo.add(s);
    }
    for (let ss of s2) {
        if (repo.has(ss)) {
            answer += 1;
        }
    }
    return answer;
}

File: 프로그래머스/0/120903. 배열의 유사도/배열의 유사도.java
import java.util.HashSet;

class Solution {
    public int solution(String[] s1, String[] s2) {
        int answer = 0;
        HashSet<String> repo = new HashSet<>();
        for (String s : s1) {
            repo.add(s);
        }
        for (String ss : s2) {
            if (repo.contains(ss)) {
                answer += 1;
            }
        }
        return answer;
    }
}